_projects/2023/100510783/100510783.Rmd

```diff
@@ -0,0 +1,277 @@
+---
+title: "With whom do we spend our time in 2020?"
+description: |
+  A short description of the post.
+categories: "2023"
+author: Eric Hausken-Brates
+date: "`r Sys.Date()`" 
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center")
+
+library(tidyverse)
```

Comment:
Please move this call to the tidyverse to another chunk down below so that the reader knows that you are loading this library.